astroquery/esa/neocc/core.py

```diff
@@ -0,0 +1,453 @@
+             2020GE 2023-09-08 00:00:00.000 ...              1.4     3.308
+            2020RS1 2023-09-10 00:00:00.000 ...              9.1     4.071
+            2023QE8 2023-09-10 00:00:00.000 ...             14.5     1.039
+                ...                     ... ...              ...       ...
+           2012SX49 2024-08-29 00:00:00.000 ...              4.3     2.665
+           2016RJ20 2024-08-30 00:00:00.000 ...             14.8     2.118
+             2021JT 2024-09-02 00:00:00.000 ...              8.3     4.216
+           2021RB16 2024-09-02 00:00:00.000 ...              8.5     3.685
+            2007RX8 2024-09-02 00:00:00.000 ...              7.0     2.322
+        Length = 176 rows
+
+        Note
+        ----
+        If the contents request fails the following message will be printed:
+
+        *Initial attempt to obtain list failed. Reattempting...*
+
+        Then a second request will be automatically sent to the NEOCC portal.
+        """
+
+        # Get URL to obtain the data from NEOCC
+        url = lists.get_list_url(list_name)
+
+        # Request list two times if the first attempt fails
+        try:
+            # Parse decoded data
+            neocc_list = lists.get_list_data(url, list_name)
+
+            return neocc_list
+
+        except ConnectionError:  # pragma: no cover
+            print('Initial attempt to obtain list failed. Reattempting...')
+            # Wait 5 seconds
+            time.sleep(5)
+            # Parse decoded data
+            neocc_list = lists.get_list_data(url, list_name)
+
+            return neocc_list
+
+    @staticmethod
+    def query_object(name, tab, **kwargs):
+        """Get requested object data from ESA NEOCC.
+
+        Parameters
+        ----------
+        name : str
+            Name of the requested object
+        tab : str
+            Name of the request tab. Valid names are: summary,
+            orbit_properties, physical_properties, observations,
+            ephemerides, close_approaches and impacts.
+        **kwargs : str
+            Tabs orbit_properties and ephemerides tabs required additional
+            arguments to work:
+
+            * *orbit_properties*: the required additional arguments are:
+
+                * *orbital_elements* : str (keplerian or equinoctial)
+                * *orbit_epoch* : str (present or middle)
+
+            * *ephemerides*: the required additional arguments are:
+
+                * *observatory* : str (observatory code, e.g. '500', 'J04', etc.)
+                * *start* : str (start date in YYYY-MM-DD HH:MM)
+                * *stop* : str (end date in YYYY-MM-DD HH:MM)
+                * *step* : str (time step, e.g. '2', '15', etc.)
+                * *step_unit* : str (e.g. 'days', 'minutes', etc.)
+
+        Returns
+        -------
+        neocc_obj : list of `~astropy.table.Table`
+            One or more tables containing the requested object data.
+            the tab selected.
+
+        Examples
```

Comment:
docstrings are superb, but this looks to be a little bit too verbose, it should not be a copy of the narrative docs covering all the corner cases, but a short showcase of how to use the method.

We also need to double-check how this all renders

Reply:
The docstrings are so extensive in the code because of how the library was intended in the first place, as a standalone version. Also to provide examples to the users. Information can b reduced if it can be added later elsewhere

Reply:
I've removed the examples as they are all in the narrative docs.